src/app/layout.tsx

```diff
@@ -7,12 +7,31 @@ import { Metadata } from "next";
 import { Inter } from "next/font/google";
 import "./globals.css";
 import { getServerSession } from "next-auth";
+import { createLogger, transports } from "winston";
+import { LoggingWinston } from "@google-cloud/logging-winston";
+
 import { getL10n, getL10nBundles } from "./functions/server/l10n";
 import { getLocale } from "./functions/universal/getLocale";
 import { SessionProvider } from "../contextProviders/session";
 import { authOptions } from "./api/utils/auth";
 import { metropolis } from "./fonts/Metropolis/metropolis";
 
+const loggingWinston = new LoggingWinston({
+  labels: {
+    name: "monitor-stats",
+    version: "0.1.0",
+  },
+});
+
+const logger = createLogger({
+  level: "info",
+  transports: [new transports.Console()],
+});
+
+if (["stage", "production"].includes(process.env.APP_ENV ?? "local")) {
+  logger.transports.push(loggingWinston);
+}
+
```

Comment:
We should consider sharing this (similar to how we share the knex config) since I think it'll be the same for every file we want to use this from.

Reply:
Yeah, I'd expect to be able to import `logger` from a module and start logging right away?

Reply:
OK cool I'll move this into a shared location. Would `functions/server` be appropriate or is there a better place? It's definitely a server-side-only thing (although we could proxy messages from the client -> API -> server anyway to make our data collection first-party, right now Glean.js submits to the `incoming.telemetry.mozilla.org` endpoint)